src/components/InstallTabs/InstallTabs.scss

```diff
@@ -105,7 +105,7 @@
       padding: var(--space-08) var(--space-16);
       position: absolute;
       bottom: 2rem;
-      right: 2rem;
+      left: 2rem;
```

Comment:
I moved the button to the bottom right which is how its described in the design. 
@MylesBorins gave a thumbs up on visual standpoint (#742)

Reply:
We can optimize it in a separate PR. It needs more fixing.